com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -635,20 +635,41 @@ internal static FbxVector4 ConvertToFbxVector4(Vector3 leftHandedVector, float u
                 return false;
             }
 
+            var offset = unityMaterial.GetTextureOffset(unityPropName);
+            var scale = unityMaterial.GetTextureScale(unityPropName);
+            var wrapModeU = unityTexture.wrapModeU;
+            var wrapModeV = unityTexture.wrapModeV;
+
+            var tuple = (unityTexture, offset, scale, wrapModeU, wrapModeV);
+            
             // Find or create an fbx texture and link it up to the fbx material.
-            if (!TextureMap.ContainsKey (textureSourceFullPath)) {
+            if (!TextureMap.ContainsKey (tuple)) {
                 var textureName = GetUniqueTextureName(fbxPropName + "_Texture");
                 var fbxTexture = FbxFileTexture.Create (fbxMaterial, textureName);
                 fbxTexture.SetFileName (textureSourceFullPath);
                 fbxTexture.SetTextureUse (FbxTexture.ETextureUse.eStandard);
                 fbxTexture.SetMappingType (FbxTexture.EMappingType.eUV);
-                TextureMap.Add (textureSourceFullPath, fbxTexture);
+                fbxTexture.SetScale(scale.x, scale.y);
+                fbxTexture.SetTranslation(offset.x, offset.y);
+                fbxTexture.SetWrapMode(GetWrapModeFromUnityWrapMode(wrapModeU), GetWrapModeFromUnityWrapMode(wrapModeV));
+                TextureMap.Add (tuple, fbxTexture);
             }
-            TextureMap [textureSourceFullPath].ConnectDstProperty (fbxMaterialProperty);
+            TextureMap [tuple].ConnectDstProperty (fbxMaterialProperty);
 
             return true;
         }
 
+        private FbxTexture.EWrapMode GetWrapModeFromUnityWrapMode(TextureWrapMode wrapMode)
+        {
+            switch (wrapMode)
+            {
+                case TextureWrapMode.Clamp: return FbxTexture.EWrapMode.eClamp;
+                case TextureWrapMode.Repeat: return FbxTexture.EWrapMode.eRepeat;
+                default:
+                    Debug.LogWarning("Texture uses wrap mode " + wrapMode + " which is not supported by FBX. Will be exported as \"Repeat\".");
```

Comment:
I'm not convinced we should issue this warning. When it comes to materials we don't claim to transfer losslessly.

If we do really want to issue a warning I'd recommend at least including a name that the user can search for to go check on maybe they want to fix it and export again. And maybe an object for context.

Reply:
will fix in another PR